resources/views/users/modals/add_group.blade.php

```diff
@@ -1,3 +1,17 @@
+<style>
+input,textarea{
+    border-bottom:2px solid #253c5c;
+    border-right:none;
+    border-left:none;
+    border-top:none;
+    width: 25rem
```

Comment:
put the styles in a css file

Reply:
Because that affects other pages i moved them into PHP file. Should I change the class name instead of using style?

Reply:
Yes, you should just use different class name instead.

Reply:
and yes, you can add a specific class or id that intends for the specific purpose of the page